nextcord/message.py

```diff
@@ -1555,6 +1555,10 @@ async def create_thread(self, *, name: str, auto_archive_duration: ThreadArchive
 
         .. versionadded:: 2.0
 
+        .. note::
+
+            This will raise :exc:`HTTPException` if the guild features isn't contain ``PRIVATE_THREADS`` and you are trying to create a private thread.
```

Comment:
```suggestion
        .. note::

            This will raise :exc:`HTTPException` if the guild features does not include ``PRIVATE_THREADS`` and you are trying to create a private thread.
```